value/src/test/java/com/google/auto/value/processor/ExtensionTest.java

```diff
@@ -120,12 +121,49 @@ public void testExtensionConsumesProperties() throws Exception {
     );
     assertAbout(javaSource())
         .that(javaFileObject)
-        .processedWith(
-            new AutoValueProcessor(ImmutableList.<AutoValueExtension>of(new FooExtension())))
+        .processedWith(new AutoValueProcessor(ImmutableList.of(new FooExtension())))
         .compilesWithoutError()
         .and().generatesSources(expectedExtensionOutput);
   }
 
+  public void testDoesntRaiseWarningOnConsumedProperties() {
+    JavaFileObject impl = JavaFileObjects.forSourceLines("foo.bar.Baz",
+        "package foo.bar;",
+        "import com.google.auto.value.AutoValue;",
+        "@AutoValue public abstract class Baz {",
+        "  abstract String foo();",
+        "  abstract String dizzle();",
+        "}");
+    assertAbout(javaSource())
+        .that(impl)
+        .processedWith(new AutoValueProcessor(ImmutableList.of(new FooExtension())))
+        .compilesWithoutError()
+        // expected warnings: @AutoValue and @Generated annotations not consumed
+        .withWarningCount(2);
```

Comment:
Er, no, `-Xlint:-processing` is only about the warning wrt annotations that haven't been _claimed_ by a processor (and most processors, including Auto ones, don't claim annotations at all).
https://docs.oracle.com/javase/7/docs/technotes/tools/solaris/javac.html#xlintwarnings